packages/elements/src/clock/index.ts

```diff
@@ -482,28 +526,58 @@ export class Clock extends ResponsiveElement {
   * @param event Event Object
   * @returns {void}
   */
-  private handleUpKey (event: KeyboardEvent): void {
-    this.shift(UP, this.getShiftAmountFromTarget(event.target));
+  private handleUpKey (): void {
+    const selection = this.renderRoot.querySelector(`[part~=${this.activeSegment}]`);
+    this.shift(Direction.UP, this.getShiftAmountFromTarget(selection));
   }
 
   /**
   * Handle DOWN key press
   * @param event Event Object
   * @returns {void}
   */
-  private handleDownKey (event: KeyboardEvent): void {
-    this.shift(DOWN, this.getShiftAmountFromTarget(event.target));
+  private handleDownKey (): void {
+    const selection = this.renderRoot.querySelector(`[part~=${this.activeSegment}]`);
+    this.shift(Direction.DOWN, this.getShiftAmountFromTarget(selection));
+  }
+
+  /**
+  * Handles Left key press
+  * @param event Event Object
+  * @returns {void}
+  */
+  private handleLeftRightKey (): void {
+    this.activeSegment = this.activeSegment === Segment.HOURS ? Segment.MINUTES : Segment.HOURS;
+  }
+
+  /**
+  * Updates aria-value-text to have the same value as screen
+  * @returns {void}
+  */
+  private updateAriaValue () {
+    const value = `Time: ${padNumber(this.displayHours, 2)}`
+      + `:${padNumber(this.displayMinutes, 2)}`
```

Comment:
Can we extract `:` separator like we did in a lot of elements ? Would that make it easier to read ?

Reply:
Also other than breaking the lines, extracting hours, minute etc. into variables is an alternative.

Reply:
Resolved by Alek PR